data/fleets.txt

```diff
@@ -1903,6 +1903,16 @@ fleet "Small Core Pirates"
 		"Sparrow"
 	variant 1
 		"Fury (Heavy)"
+	variant 2
+		"Wasp (Gatling)"
+	variant 2
+		"Wasp (Javelin)"
+	variant 2
+		"Wasp (Meteor)"
+	variant 2
+		"Wasp (Plasma)"
+	variant 2
+		"Wasp (Proton)"
```

Comment:
Fleet | Old chance | New chance
------|--------------|--------------
Single interceptor | 65.5% | 74.4%
Double interceptor | 6.9% | 5.1%
Light warship | 27.6% | 20.5%

Core pirate fleets might lean too much toward single interceptor fleets now. I'd suggest either reducing the frequency of all these wasp variants to 1, or adding new double interceptor and light warship variants to compensate and keep the chance of each fleet variant close to the original.

Reply:
New changes:

    single Quicksilver from 4 to 6
    double Quicksilver from 2 to 3
    single Wasp variants from 0 to 10
    removed interceptor variants introduced in #3922 because some Wasps have missiles; from 5 to 0

That ought to largely preserve average fleet strength, I think.